tests/test_model_schema.py

```diff
@@ -722,11 +722,13 @@ def test_deserialization_of_seminar_with_many_lectures_that_DNE(
             "semester": "First",
             "lectures": [
                 {
+                    "id": 99999998,
                     "topic": "Intro to Ter'Angreal",
                     "seminar_title": "Novice Training",
                     "seminar_semester": "First",
                 },
                 {
+                    "id": 99999999,
                     "topic": "History of the Ajahs",
                     "seminar_title": "Novice Training",
                     "seminar_semester": "First",
```

Comment:
Including the `"id"` keys here silences a sqlalchemy warning about retrieving an object without primary key.